src/components/screens/dashboard/dashboard.js

```diff
@@ -61,7 +61,7 @@ const Dashboard = ({ account, t }) => {
         />
         <div className={`${styles.main}`}>
           <div className={styles.subContainer}>
-            {isLoggedIn ? <WalletDetails className={styles.marginFix} /> : null}
+            {isLoggedIn ? <WalletDetails className={styles.marginFix} isWalletRoute={false} /> : null}
```

Comment:
Here you have a linting issue 
```suggestion
            {
              isLoggedIn
                ? <WalletDetails className={styles.marginFix} isWalletRoute={false} />
                : null
            }
```